### DISCUSSION TITLE
Is it possible to allow wildcards for mapping a GoCardless import?

### DISCUSSION THREAD

**Discussion by Jademalo** [firefly-iii/firefly-iii#8609] [Firefly III]
After importing my data through GoCardless, I've realised I could do with wildcard maps for future automation.

For example, any transaction from PayPal is `PAYPAL *xxxx`. I want all of these to map as transfers from my account to PayPal, regardless of what the specific reference from PayPal is. 
While I can manually map transactions, I can't figure out a way to automatically map everything that starts with PayPal. 

Is this possible?

**Comment by JC5** [comment]
No, sorry. You can use webhooks to do this yourself though.

**Reply by Jademalo** [reply]
Alright, thank you. I've realised I can get the same result with Rules too, which is much simpler.
